app/src/main/res/layout/welcome.xml

```diff
@@ -34,7 +34,7 @@
         android:layout_marginEnd="@dimen/home_welcome_text_view_margin_end"
         android:fontFamily="sans-serif"
         android:text="@{viewModel.greeting}"
-        android:textColor="@color/component_color_home_activity_greeting_text_color"
+        android:textColor="@color/component_color_shared_content_item_text_view_color"
```

Comment:
Remove this and use correct color `component_color_shared_content_item_text_view_color`

Reply:
Done.